dockstore-webservice/src/test/resources/badVersionCWL.cwl

```diff
@@ -0,0 +1,25 @@
+cwlVersion: v2.7
```

Comment:
just in case, change to something more ridiculous like 9001.9001

Reply:
Increased ridiculousness [here](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/resources/badVersionCWL.cwl#L1).